src/test/groovy/ratpack/zipkin/ServerTracingModuleSpec.groovy

```diff
@@ -265,8 +261,8 @@ class ServerTracingModuleSpec extends Specification {
 		then:
 			reporter.getSpans().size() == 1
 			Span span = reporter.getSpans().get(0)
-			span.binaryAnnotations.find { it -> it.key == TraceKeys.HTTP_STATUS_CODE } != null
-			span.binaryAnnotations.find { it -> it.key == Constants.ERROR } != null
+			span.tags().find { it -> it.key == TraceKeys.HTTP_STATUS_CODE } != null
```

Comment:
might be a map assertion?

Reply:
Thanks for the suggestion!  Map assertions are awesome... I clearly don't use assertj enough :)